Src/CSharpier/CodeFormatter.cs

```diff
@@ -150,7 +156,7 @@ private string PrintDocTree(Doc document, string indent)
                                 ? "HardLine"
                                 : "SoftLine");
                 case BreakParent breakParent:
-                    return null;
+                    return "";
```

Comment:
I switched these all to string.Empty mostly because that's what we standardized at my work. I'm not opposed to switching em all to "" if you have strong feelings.

Reply:
Let's standardize on `string.Empty`. I wonder if that's something we can make csharpier do automatically :thinking: